xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/DownloadAction.java

```diff
@@ -82,6 +82,11 @@
     /** Key of the blacklist in xwiki.properties. */
     public static final String BLACKLIST_PROPERTY = "attachment.download.blacklist";
 
+    /**
+     * Key of the forceDownload list in xwiki.properties
+     */
+    public static final String FORCE_DOWNLOAD_PROPERTY = "attachment.download.forceDownload";
```

Comment:
Missing since annotation. I guess unstable would make sense too.

Reply:
Indeed. I guess I didn't get complains from revapi because I didn't try to build on legacy.

Reply:
There is not reason for revapi to complain about a new static field.